### DISCUSSION TITLE
input with MTD produces output with MT?

### DISCUSSION THREAD

**Discussion by armandlcz** [FeynCalc/feyncalc#227] [Q&A]
Hi everyone, 

first of all I want to thank all of you feyncalc team because of the awesome work you do. I wonder if you can help me with the following issue I have. I am doing a computation in dimension D, the starting expression (input) involves only MTD[\[Mu], \[Nu]] (not MT[\[Mu], \[Nu]]) but the result I get (output) is written in terms of MTD[\[Mu], \[Nu]] and MT[\[Mu], \[Nu]] ( mixing 4-dimensional metric \bar{g}^{\mu\nu} and D-dimensional metric g^{\mu\nu} without bar). I have checked all the input expressions and all of them are written only in terms of MTD[\[Mu], \[Nu]] (not MT[\[Mu], \[Nu]]). I traced back this issue and I arrived at this comparative example:

Input: MTD[0, b] // FCE // StandardForm
Output: MTD[0, b]-------->OK,

on the other hand:
Input: Contract[MTD[0, b] MTD[a, a]] // FCE // StandardForm
Output: D MT[0, b]-------->???: why MT[0,b] instead of MTD[0,b]? it should appear MTD[0,b] souldn't it?. This example seems to be the cause of the mixing between MTD and MT I get.

I would be grateful if you can help me understand what is going on.
FeynCalc 9.3.1 (stable version)
Wolfram Mathematica 12.0.0.0.

**Comment by vsht** [comment]
Hi,

the conventional dimensional regularization does not modify the temporal component of Lorentz tensors. Only the 
spatial components are transferred from $3$ to $D-1$ dimensions. Hence, any tensor having explicit $0$-components
is automatically treated as 4-dimensional in FeynCalc.

**Reply by armandlcz** [reply]
Thanks Vlad!
